SpiffWorkflow/bpmn/parser/BpmnParser.py

```diff
@@ -160,12 +161,25 @@ def add_bpmn_xml(self, bpmn, filename=None):
         for process in xpath('.//bpmn:process'):
             self.create_parser(process, xpath, filename)
 
+        self._find_dependencies(xpath)
+
         collaboration = first(xpath('.//bpmn:collaboration'))
         if collaboration is not None:
             collaboration_xpath = xpath_eval(collaboration)
             name = collaboration.get('id')
             self.collaborations[name] = [ participant.get('processRef') for participant in collaboration_xpath('.//bpmn:participant') ]
 
+    def _find_dependencies(self, xpath):
+        """Locate all calls to external BPMN and DMN files, and store their
+        ids in our list of dependencies"""
+        for call_activity in xpath('.//bpmn:callActivity'):
+            self.dependencies.append(call_activity.get('calledElement'))
```

Comment:
Not sure if you can get into a situation where you have duplicate dependencies, like multiple callActivity nodes that have the same calledElement? If so, perhaps a set for the dependencies?

Reply:
excellent point, I'll assure it's a unique set.